runtime/j9vm31/exports.cmake

```diff
@@ -0,0 +1,265 @@
+################################################################################
+# Copyright (c) 2021, 2021 IBM Corp. and others
+#
+# This program and the accompanying materials are made available under
+# the terms of the Eclipse Public License 2.0 which accompanies this
+# distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+# or the Apache License, Version 2.0 which accompanies this distribution and
+# is available at https://www.apache.org/licenses/LICENSE-2.0.
+#
+# This Source Code may also be made available under the following
+# Secondary Licenses when the conditions for such availability set
+# forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+# General Public License, version 2 with the GNU Classpath
+# Exception [1] and GNU General Public License, version 2 with the
+# OpenJDK Assembly Exception [2].
+#
+# [1] https://www.gnu.org/software/classpath/license.html
+# [2] http://openjdk.java.net/legal/assembly-exception.html
+#
+# SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+################################################################################
+
+omr_add_exports(jvm31
+	JNI_CreateJavaVM
+	JNI_GetCreatedJavaVMs
+	JNI_GetDefaultJavaVMInitArgs
+	DestroyJavaVM
+	AttachCurrentThread
```

Comment:
Rather than repeat all the exports, I think we should refactor things so `j9vm` and `j9vm31` can share (perhaps some conditionally).

Reply:
On the other hand, perhaps I misunderstand the meaning of this file?

Reply:
The names are quite different between j9vm and j9vm31 -- what we export is based on the implementation names.  For libjvm31, we used the actual JNI function names.  I expect JNI functions are generally invoked via the JavaNativeInterface_ function table anyway.